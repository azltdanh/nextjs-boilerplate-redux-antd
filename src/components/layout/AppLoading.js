import { Spin } from 'antd';

const AppLoading = () => (
  <div className="app-loading">
    <Spin />
  </div>
);

export default AppLoading;
