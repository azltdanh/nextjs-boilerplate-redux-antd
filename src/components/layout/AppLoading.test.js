import AppLoading from './AppLoading';

describe(AppLoading.name, () => {
  it('renders', () => {
    const wrapper = shallow(<AppLoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
