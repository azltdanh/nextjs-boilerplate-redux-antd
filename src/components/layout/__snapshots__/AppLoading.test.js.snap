// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoading renders 1`] = `
<div
  className="app-loading"
>
  <Spin
    size="default"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
